src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageData.java

```diff
@@ -25,92 +48,1243 @@
     public String groupByTeam = null;
     public String showStats = null;
     public int startIndex;
+    private boolean shouldCollapsed;
+    
+    private FieldValidator validator = new FieldValidator();
+
 
     // used for html table ajax loading
     public String courseId = null;
     public String feedbackSessionName = null;
     public String ajaxStatus = null;
     public String sessionResultsHtmlTableAsString = null;
+    
 
+    // TODO multiple page data classes for each view type
+    
+    // for question view
+    List<InstructorResultsQuestionTable> questionPanels;
+    // for giver > question > recipient, recipient > question > giver...
+    LinkedHashMap<String, InstructorFeedbackResultsSectionPanel> sectionPanels;
```

Comment:
Hmm, it may be good to put a TODO to convert that into some sort of class in a sorted list, then, considering how Map doesn't seem to be the right structure for it.